server/requestlog/ncsa.go

```diff
@@ -59,27 +59,29 @@ func (l *NCSALogger) log(ent *Entry) error {
 
 func formatEntry(b []byte, ent *Entry) []byte {
 	const ncsaTime = "02/Jan/2006:15:04:05 -0700"
-	if ent.RemoteIP == "" {
+	entReq := ent.Request
```

Comment:
Why are you changing this and the "Entry" type? I don't understand what that has to do with OpenTelemetry.